### ISSUE TITLE
Have to click twice to resize window

### ISSUE THREAD

**Issue by Jinglespell** [leocb/MaterialSkin#116]
Have an issue where I have to click twice to resize the main window, one time to begin drag and one time to release, which feels uncomfortable. 

Maybe am I missing something ?

**Comment by leocb** [comment]
Be sure to leave ~5 pixels gap at the bottom and sides of the form before any control, this makes it easier to grab the portion of the form to resize, also, due to the hacky way this is implemented on a borderless form, the implementation is not 100% and has some bugs, see #99 and #75
Also, Maybe you're resizing too quickly? Try to give the interface some time to redraw everything by waiting a little bit after you resize before releasing the mouse

**Comment by Jinglespell** [comment]
Hi! Thanks for answering that fast. Setting MaterialForm.Sizable to false seems to solve this for reasons. Fairly new with MaterialSkin so I don't know if this is intended

**Comment by leocb** [comment]
Yes that's correct, I guess I could enforce that on the code itself, so users don't have to do this by hand.
Thanks for reporting!
